docs/docsgen/source/api/checker.rst

```diff
@@ -5,6 +5,36 @@ onnx.checker
 .. contents::
     :local:
 
+check_value_info
```

Comment:
`C.CheckerContext`  should be exposed as well.

Reply:
Just exposes it. Not sure whether the usage is correct though. PTAL. Thanks!